static/app/components/projects/appStoreConnectContext/utils.tsx

```diff
@@ -1,17 +1,20 @@
-import {t} from 'app/locale';
-import {AppStoreConnectValidationData} from 'app/types/debugFiles';
+import {
+  getAppStoreValidationErrorMessage,
+  ValidationErrorDetailed,
+} from 'app/components/modals/debugFileCustomRepository/appStoreConnect/utils';
+import {AppStoreConnectCredentialsStatus} from 'app/types/debugFiles';
 
-export const appStoreConnectAlertMessage = {
-  appStoreCredentialsInvalid: t(
-    'The credentials of your configured App Store Connect are invalid.'
-  ),
-};
+export function areAppStoreConnectCredentialsValid(
+  credentialsStatus: AppStoreConnectCredentialsStatus | undefined
+): boolean {
+  return credentialsStatus?.status === 'valid';
+}
```

Comment:
i'm not sure if this belongs here, open to suggestions on better places to put this helper